spec/factories/tags.rb

```diff
@@ -0,0 +1,5 @@
+FactoryGirl.define do
+  factory :tag do
+    name "test_tag"
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols.